packages/raystack/table/FilteredChip.tsx

```diff
@@ -29,26 +30,43 @@ interface FilterValuesProps {
   columnType?: columnTypes;
   onValueChange?: (value: FilterValue) => void;
   options?: TableColumnMetadata[];
+  filterOperation?: FilterOperation;
 }
 
 const FilterValues = ({
   columnType = filterValueTypeMap.text,
   options = [],
   onValueChange,
+  filterOperation,
+  ...props
 }: FilterValuesProps) => {
-  const [value, setValue] = useState("");
-  const valueType: filterValueType =
-    columnType === columnTypesMap.select
-      ? filterValueTypeMap.select
-      : filterValueTypeMap.text;
+  const [value, setValue] = useState<FilterValue>({
+    date: new Date(),
+    dateRange: {
+      to: new Date(),
+      from: new Date(),
+    },
+  });
+
+  const valueType: filterValueType = filterOperation?.component
```

Comment:
Can we use an if..else here? Nested ternary operations can be difficult to debug and maintain later on when we look at it after a while.